### ISSUE TITLE
Travis Update

### ISSUE THREAD

**Issue by adamcaudill** [adamcaudill/libsodium-net#118]
See here: http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade

**Comment by bitbeans** [comment]
I think you need sudo support to compile libsodium?

**Comment by adamcaudill** [comment]
Yeah, as it stands today - I think there may be a way around it, though could make the parts of the Travis setup more complex.

I'll play with it when I have a chance, but free time has been extremely limited recently, so might be a bit.
